src/main/java/duke/component/Ui.java

```diff
@@ -0,0 +1,111 @@
+package duke.component;
+
+import java.util.Scanner;
+
+import duke.task.Task;
+
+import static duke.constant.Message.GOOD_BYE;
+import static duke.constant.Message.GREETING;
+import static duke.constant.Message.HORIZONTAL_LINE;
+import static duke.constant.Message.LINE_PREFIX;
+import static duke.constant.Message.LINE_SEPARATOR;
+import static duke.constant.Message.CONFIRM_MARK;
+import static duke.constant.Message.CONFIRM_UNMARK;
```

Comment:
I really like your use of constants as it helps to organise your code much better!